snake.js

```diff
@@ -36,8 +46,8 @@ class Snake {
 
     foodEat() {
         if (this.snakeXdelta === this.foodview.x && this.snakeYdelta === this.foodview.y) {
-            this.game.food.randomfood();
-            this.game.score.score++;
+            this.food.randomFood();
+            this.score.score++;
```

Comment:
@cr7hardwork create a incrementScroeMethod and call it from ither classes instead of incrementing score directly.